source/configuration_manual/sieve/configuration.rst

```diff
@@ -0,0 +1,692 @@
+   filesystem path pointing to a script file, or - when
+   :ref:`ManageSieve <pigeonhole_managesieve_server>`
+   is used - it is the location of the symbolic link pointing to the
+   active script in the storage directory. That storage directory is
+   then configured using the deprecated :ref:`plugin-sieve-setting-sieve_dir`  setting.
+
+:ref:`plugin-sieve-setting-sieve_default` = (v0.3+)
+   The location of the default personal sieve script file which gets
+   executed ONLY if user's private Sieve script does not exist, e.g.
+   ``file:/var/lib/dovecot/default.sieve`` (check the :ref:`multiscript
+   section <pigeonhole_configuration_multiscript>` for instructions on running global Sieve
+   scripts before and after the user's personal script). This is usually
+   a global script, so be sure to pre-compile the specified script
+   manually in that case using the ``sievec`` command line tool, as
+   explained
+   `here <https://wiki2.dovecot.org/Pigeonhole/Sieve/Usage#scriptcompile>`__.
+   This setting used to be called :ref:`plugin-sieve-setting-sieve_global_path` , but that name
+   is now deprecated.
+
+:ref:`plugin-sieve-setting-sieve_default_name` = (v0.4.8+)
+   The name by which the default Sieve script is visible to
+   :ref:`ManageSieve <pigeonhole_managesieve_server>`
+   clients. Normally, it is not visible at all. Refer to the :ref:`visible
+   default script section <pigeonhole_configuration_visible_default_script>` for more
+   information.
+
+:ref:`plugin-sieve-setting-sieve_global`  =
+   Location for :global include scripts for the Sieve `include
+   extension <http://tools.ietf.org/html/draft-ietf-sieve-include-05>`__.
+   This setting used to be called :ref:`plugin-sieve-setting-sieve_global_dir`, but that name is
+   now deprecated.
+
+:ref:`plugin-sieve-setting-sieve_discard` = (v0.4.16+)
+   The location of a Sieve script that is run for any message that is
+   about to be discarded; i.e., it is not delivered anywhere by the
+   normal Sieve execution. This only happens when the "implicit keep" is
+   canceled, by e.g. the "discard" action, and no actions that deliver
+   the message are executed. This "discard script" can prevent
+   discarding the message, by executing alternative actions. If the
+   discard script does nothing, the message is still discarded as it
+   would be when no discard script is configured.
+
+:ref:`plugin-sieve-setting-sieve_extensions`  =
+   Which Sieve language extensions are available to users. By default,
+   all supported extensions are available, except for deprecated
+   extensions, extensions that add the ability to change messages,
+   extensions that require explicit configuration or extensions that are
+   still under development. Some system administrators may want to
+   disable certain Sieve extensions or enable those that are not
+   available by default. All supported extensions are listed
+   `here <https://wiki2.dovecot.org/Pigeonhole/Sieve#features>`__.
```

Comment:
this shouldn't link to the wiki (probably missed this one last time. reviewing this is quite terrible).